pkg/audit/manager.go

```diff
@@ -600,6 +610,15 @@ func (am *Manager) reviewObjects(ctx context.Context, kind string, folderCount i
 			}
 
 			if len(resp.Results()) > 0 {
+				if *logStatsAudit {
```

Comment:
Don't we want to log stats even if no violations were thrown? This also matches the behavior of the audit-from-cache code above.

Reply:
Ah yes, this was an oversight on my part!